src/test/java/edu/kis/powp/jobs2d/TestJobs2dApp.java

```diff
@@ -60,6 +61,10 @@ private static void setupCommandTests(Application application) {
 
         application.addTest("Run command", new SelectRunCurrentCommandOptionListener(DriverFeature.getDriverManager()));
 
+        Canvas A4 = new Canvas(210,297);
+        Canvas A7 = new Canvas(74,105);
```

Comment:
You could make a factory for easier Canvases reuse.

Reply:
fixed